mappers/bluetooth_mapper/README.md

```diff
@@ -0,0 +1,318 @@
+# Bluetooth Mapper
+
+
+## Introduction
+
+Mapper is an application that is used to connect and control devices. This is an implementation of mapper for 
+bluetooth protocol. The aim is to create an application through which users can easily operate devices using bluetooth protocol for communication to the KubeEdge platform. The user is required to provide the mapper with the information required to control their device through the configuration file these can be changed at runtime by providing the input through the MQTT broker.
+
+## Running the mapper
+
+While running the mapper please keep in mind the following points :-
+
+   1. Please run the binary as root or with root permissions.
+   2. Please ensure that bluetooth service of your device is ON
+   3. Please remember to give one of the following command line parameters for the logging service:
+    
+     -logtostderr=true      	   // Logs are written to standard error instead of to files.
+     -alsologtostderr=true         // Logs are written to standard error as well as to files.
+     -stderrthreshold=ERROR		   // Log events at or above this severity are logged to standard error as well as to files.
+     -log_dir=""          		   // Log files will be written to this directory instead of the default temporary directory.
+
+## Modules
+
+There are five modules in this implementation of bluetooth mapper, namely :
+ 1. Action Manager
```

Comment:
I saw the design of mapper.
https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/mapper-design.md#proposal
I find `Event-Manager` is in the architecture diagram.
Is the same module for `Action Manager` and `Event-Manager` ?

Reply:
In the initial draft of the mapper design, it was named as Event-Manager but after review the name was changed to Action Manager, i guess the name in the architecture diagram has not been corrected yet. @sids-b Can u pls update the name of Event Manager in the architecture diagram ?

Reply:
Sure, will do it